jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseRuntimeDataServiceImplTest.java

```diff
@@ -89,7 +89,7 @@ public void testStartEmptyCaseWithCaseFile() {
             assertNotNull(cInstance.getCaseFile());
             assertEquals("my first case", cInstance.getCaseFile().getData("name"));
 
-            Collection<CaseInstance> instances = caseRuntimeDataService.getCaseInstances(new QueryContext());
+            Collection<CaseInstance> instances = caseRuntimeDataService.getCaseInstances(Arrays.asList(CaseStatus.OPEN), true, new QueryContext());
```

Comment:
As there is only 1 single list as argument, it could be replaced with:

```suggestion
            Collection<CaseInstance> instances = caseRuntimeDataService.getCaseInstances(Collections.singletonList(CaseStatus.OPEN), true, new QueryContext());
```

Reply:
yes, thats better